common/test/src/View/ChangeBrushFaceAttributesTest.cpp

```diff
@@ -110,5 +110,117 @@ namespace TrenchBroom {
             document->redoCommand();
             checkTexture("texture2");
         }
+
+        TEST_CASE_METHOD(ChangeBrushFaceAttributesTest, "ChangeBrushFaceAttributesTest.setAll") {
+            Model::BrushNode* brushNode = createBrushNode();
+            document->addNode(brushNode, document->currentParent());
+            const Model::Brush& brush = brushNode->brush();
```

Comment:
I believe the `brush` and `*FaceAttributes` variables here need to be non-references, because the Brush gets swapped out when `document->setFaceAttributes(..)` is executed. Might explain the test failures?